openml/datasets/dataset.py

```diff
@@ -368,9 +368,18 @@ def decode_arff(fh):
     def _convert_array_format(data, array_format, attribute_names):
         """Convert a dataset to a given array format.
 
-        By default, the data are stored as a sparse matrix or a pandas
-        dataframe. One might be interested to get a pandas SparseDataFrame or a
-        NumPy array instead, respectively.
+        Parameters
+        ----------
+        array_format : str
+            Tag to attach to the dataset to get a pandas SparseDataFrame or a
+            NumPy array instead.
+            - If array_format='array'
+                Converts non-sparse numeric data to numpy-array
+                Enforces numeric encoding of categorical columns
+                Missing values are represented as NaN in the dataframe
```

Comment:
I think you mean `ndarrray` instead of `dataframe`?

Reply:
Oh yes, my bad.